invisible_cities/io/mcinfo_io.py

```diff
@@ -168,6 +172,37 @@ def __call__(self, mctables: (tb.Table, tb.Table, tb.Table, tb.Table),
         self.generator_table.flush()
 
 
+def copy_mc_info(files_in, h5out, event_list):
+    """
+    Copies MC info for selected events to an output file.
+    files_in :   List[str]
+                 List with the name of the input files.
+    file_out   : str
+                 The name of the output file.
+    event_list : List[int]
+                 List of events (event_id) to be copied.
+    """
+
+    writer = mc_info_writer(h5out)
+
+    for f in files_in:
+        try:
+            with tb.open_file(f, "r") as h5in:
+                run_number = get_run_number(h5in)
+                mc_info    = get_mc_info_safe(h5in, run_number)
+                for n in event_list[:]:
+                    try:
+                        writer(mctables=mc_info, evt_number=n)
+                        # We succeeded copying the event. Remove its id
+                        # from the event list
+                        event_list.remove(n)
+                    except IndexError:
+                        # The event id was not found in the current file
+                        continue
+        except:
```

Comment:
catch specific exceptions and, in case they are too generic, put some comment about what you are catching.